Makefile

```diff
@@ -30,7 +30,7 @@ setup: install-piptools ## Install requirements
 # Warning: this will install the requirements in your system python
 .PHONY: setup-global-uv
 setup-global-uv:
-	uv pip install --system -r dev-requirements.in
+	SETUPTOOLS_SCM_PRETEND_VERSION="3.0.0.dev0" uv pip install --system -r dev-requirements.in
```

Comment:
I'm still confused. Why are we seeing this now and now when I introduced uv in https://github.com/flyteorg/flytekit/pull/2403?

Reply:
I'm curious about the reason too.

And after adding `SETUPTOOLS_SCM_PRETEND_VERSION`, it indeed fixed my broken CI.

Reply:
feel like the regression in UV. they just released yesterday. https://pypi.org/project/uv/#history

Reply:
This is unfortunate. I couldn't find any issue on [their](https://github.com/astral-sh/uv/issues) tracker though.